### PR TITLE
feat: Add "On Rent" section listing rentals

### LINKED ISSUE #749: On Rent page inside My Assets
Display a new section called on rent that will show the state of rented assets.

### PR DESCRIPTION
**Pull request by juanmahidalgo**
Closes #749

image

### CODE REVIEW

**Comment by cazala** [comment]
Is it okay that clicking on a parcel takes you to the detail page, instead of the manage page?

**Review by LautaroPetaccio — CHANGES_REQUESTED** [review]

**Review by cazala — COMMENTED** [review]
Is it okay that clicking on a parcel takes you to the detail page, instead of the manage page?

**Comment by LautaroPetaccio** [comment]
It's not @cazala , great catch!

File: webapp/src/modules/ui/browse/selectors.ts [hunk 1]
```diff
@@ -64,6 +65,25 @@ export const getOnSaleNFTs = createSelector<
...
 )
 
+export const getOnRentNFTs = createSelector<
+  RootState,
+  ReturnType<typeof getAddress>,
+  ReturnType<typeof getNFTData>,
+  ReturnType<typeof getRentalData>,
+  OnRentNFT[]
+>(getAddress, getNFTData, getRentalData, (address, nftsById, rentalsById) =>
+  Object.values(nftsById)
```
[the remark this answers is not in the document]
Comment (juanmahidalgo) [reply webapp/src/modules/ui/browse/selectors.ts:75]: sure! tests added for the whole file :)

File: webapp/src/components/OnSaleOrRentList/OnSaleOrRentList.tsx [hunk 2]
```diff
@@ -8,13 +8,16 @@ import {
...
 import { useProcessedElements } from './utils'
-import styles from './OnSaleList.module.css'
+import OnSaleListElement from './OnSaleListElement'
+import OnRentListElement from './OnRentListElement'
+import styles from './OnSaleOrRentList.module.css'
 
-const OnSaleList = ({ elements, isLoading }: Props) => {
+const OnSaleOrRentList = ({ elements, isLoading, onSaleOrRentType }: Props) => {
+  const showRents = onSaleOrRentType === OnSaleOrRentType.RENT
+  console.log('elements: ', elements)
```
[the remark this answers is not in the document]
Comment (juanmahidalgo) [reply webapp/src/components/OnSaleOrRentList/OnSaleOrRentList.tsx]: done!

**Comment by juanmahidalgo** [comment]
> Is it okay that clicking on a parcel takes you to the detail page, instead of the manage page?

Updated!

**Review by LautaroPetaccio — APPROVED** [review]

**Pull request merged**